force-app/tests/Async Apex Recipes/ScheduledApexRecipes_Tests.cls

```diff
@@ -19,19 +19,19 @@ private inherited sharing class ScheduledApexRecipes_Tests {
             WHERE id = :scheduledJobId
             LIMIT 1
         ];
-        System.assertEquals(
+        System.Assert.areEqual(
```

Comment:
Fix param order in this test and the ones below

Reply:
Fixed!